ChangeLog

```diff
@@ -33,6 +33,8 @@ Bugfix
    * Fix mbedtls_x509_crt_profile_suiteb, which used to reject all certificates
      with flag MBEDTLS_X509_BADCERT_BAD_PK even when the key type was correct.
      In the context of SSL, this resulted in handshake failure. #1351
+   * Fix test suite test_suite_pk test pk_rsa_overflow passes valid parameters 
+     for hash and sig. #1179
```

Comment:
We use complete English sentences in the changelog, and it isn't just `pk_rsa_overflow`. Suggestion: “In test_suite_pk, pass otherwise valid parameters when testing hash length overflow. \#1179”